fiat-core/fiat-core.gradle

```diff
@@ -1,5 +1,6 @@
 dependencies {
 
+  implementation "net.logstash.logback:logstash-logback-encoder:4.11"
```

Comment:
I think this should leave the version unspecified and use the version from `kork` rather than specifying a specific version.

Reply:
Hi Eric, as the issue itself explians that doesnt support structured logging, fiat cant take kork dependency like all other service. Hence we specified the logger with version.

Reply:
I understand that you need to pull in the `logstash-logback-encoder` to fix this issue, but my question is why you are explicitly specifying the version in `fiat-core.gradle`.  In general (as you'll see for other dependencies in that file) we leave the version unspecified and rely on the versions specified in [spinnaker-dependencies](https://github.com/spinnaker/kork/blob/master/spinnaker-dependencies/spinnaker-dependencies.gradle) in `kork` to actually set the version.

Reply:
Ok.. will update the fiat-core.gradle without version and submit new PR for this. Thanks